### ISSUE TITLE
variants of products in pack of products

### ISSUE THREAD

**Issue by D2Red** [PrestaShop/PrestaShop#11587]
**the error**
if you want to create a pack of products, and a single product, you want to select, has more than 30 variants, only the first 30 variants are displayed in the selection.(pic: pop_1.png)
![pop_1](https://user-images.githubusercontent.com/45538290/49341700-8fca8c00-f651-11e8-8b97-d4d357106c22.png)

is there a way to show more than 30 Variants of a product?

**Additionnal information**
PrestaShop version: 1.7.4.0
PHP version: 7.1

**Comment by khouloudbelguith** [comment]
Hi @D2Red,

Thanks for your report.
I manage to reproduce the issue with PS1.7.4.4 / 1.7.5.0beta1.
For The Product P1 we have more than 30 combinations.
![image](https://user-images.githubusercontent.com/16067358/49367368-29467c00-f6ec-11e8-9c06-7436dddfe45b.png)
But, when we create a pack with this product only 30 combinations are displayed.
![image](https://user-images.githubusercontent.com/16067358/49367335-1338bb80-f6ec-11e8-80a7-1b9c0b9b87b0.png)
We will see how to fix it.
Thanks!

**Comment by marionf** [comment]
@khouloudbelguith if you enter the combination that you search, you can select it ?

**Comment by khouloudbelguith** [comment]
@marionf, no, I can just search with the name of the product, I tried with reference of the combination but it is not displayed.
https://drive.google.com/file/d/15Kaz8O2l987vXwIK0V3TeV4bj9UbcDyO/view
THanks!

**Comment by D2Red** [comment]
as khouloudbelguith wrote, only if you give the full name of the product. but some names are very long

**Comment by NANOmatic** [comment]
I have exactly the same issue. Has problem been resolved?

**Comment by MatShir** [comment]
related to https://github.com/PrestaShop/PrestaShop/issues/26566
